### PR TITLE
Modified Response descriptions for DomainTools Iris Investigate and Iris Enrich connectors

### PR DESCRIPTION
**Pull request by RamboV**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json [hunk 1]
```diff
@@ -761,14 +806,14 @@
...
-                                                        "description": "Admin Contact Name",
+                                                        "description": "Name of the administrative contact",
                                                         "title": "Admin Contact Name"
                                                     },
                                                     "count": {
                                                         "type": "integer",
                                                         "format": "int32",
-                                                        "description": "Count",
-                                                        "title": "Count"
+                                                        "description": "Count of connected domains sharing admin contact name",
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json:815]: @RamboV please make sure everything is grammatically correct. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json [hunk 2]
```diff
@@ -778,14 +823,14 @@
                                                 "properties": {
                                                     "value": {
                                                         "type": "string",
-                                                        "description": "Admin Contact Org",
+                                                        "description": "Organization of the administrative contact",
                                                         "title": "Admin Contact Org"
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json:827]: @RamboV please make sure that the sentences are complete and are grammatically correct.
Make sure that English language strings are in complete form.

File: certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json [hunk 3]
```diff
@@ -795,14 +840,14 @@
...
+                                                        "description": "Street address of the administrative contact",
                                                         "title": "Admin Contact Street"
                                                     },
                                                     "count": {
                                                         "type": "integer",
                                                         "format": "int32",
-                                                        "description": "Count",
-                                                        "title": "Count"
+                                                        "description": "Count of connected domains sharing admin contact street",
+                                                        "title": "Admin Contact Street Count"
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json:850]: @RamboV please make sure that the sentences are complete and are grammatically correct.
Make sure that English language strings are in complete form. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json [hunk 4]
```diff
@@ -829,14 +874,14 @@
                                                 "properties": {
                                                     "value": {
                                                         "type": "string",
-                                                        "description": "Admin Contact State",
+                                                        "description": "State or province of the administrative contact",
                                                         "title": "Admin Contact State"
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json:878]: @RamboV please look through the changes and make sure that strings are in complete form and everything is clear.

File: certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json [hunk 5]
```diff
@@ -1593,8 +1644,8 @@
                                                         "count": {
                                                             "type": "integer",
                                                             "format": "int32",
-                                                            "description": "Count",
-                                                            "title": "Count"
+                                                            "description": "Count of connected domains sharing ip address",
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json:1647]: @RamboV if you use abbreviations, please make sure to capitalize them. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @RamboV! Please check comments after the review. Let me know if you have any questions.
Thank you!

**Comment by RamboV** [comment]
Hello @v-EgorKozhadei ,
Thanks for the feedback, I have attended all the modifications requested. Please kindly review and provide your feedback.

File: certified-connectors/DomainToolsIrisEnrich/apiProperties.json [hunk 6]
```diff
@@ -1,56 +1,82 @@
...
-          "x-ms-apimTemplateParameter.name": "api_key",
-          "x-ms-apimTemplateParameter.value": "@connectionParameters('password')",
-          "x-ms-apimTemplateParameter.existsAction": "override"
-        }
-      }
-    ],
-    "publisher": "DomainTools, LLC",
-    "stackOwner": "DomainTools Integrations"
-  }
+    "properties": {
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisEnrich/apiProperties.json:2]: @RamboV please check the indentation in this file, it looks like contents of the file moved a bit.

**Comment by RamboV** [comment]
Hello @v-EgorKozhadei , Thanks again for your feedback, I have fixed the issues , Please check and let me know all is fixed.

File: certified-connectors/DomainToolsIrisInvestigate/apiProperties.json [hunk 7]
```diff
@@ -78,4 +78,3 @@
     "publisher": "DomainTools, LLC",
     "stackOwner": "DomainTools Integrations"
   }
-}
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiProperties.json:81]: @RamboV this curly brace is needed here, please do not delete it.

File: certified-connectors/DomainToolsIrisEnrich/apiProperties.json [hunk 8]
```diff
@@ -53,4 +53,3 @@
     "publisher": "DomainTools, LLC",
     "stackOwner": "DomainTools Integrations"
   }
-}
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisEnrich/apiProperties.json:56]: @RamboV this curly brace is needed here, please do not delete it.

File: certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json [hunk 9]
```diff
@@ -1631,8 +1682,8 @@
                                                         "count": {
                                                             "type": "integer",
                                                             "format": "int32",
-                                                            "description": "Count",
-                                                            "title": "Count"
+                                                            "description": "Count of connected domains sharing the IP country_code",
```
Comment (v-EgorKozhadei) [inline certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json:1685]: @RamboV should it be `country code` here or do you want it to be `country_code`?

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @RamboV! Thank you very much for all the fixes! There are a few minor changes left, please take a look at the comments. We're ready to go to the next phase after this.
Let me know if you have any questions, thank you!

**Comment by RamboV** [comment]
Hello @v-EgorKozhadei , Thanks a lot for the feedback. I have attended the latest 3 issues raised. Please kindly review.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hi @RamboV!

Your PR is approved and merged. Please update your submission in ISV Studio by using the latest commit ID.

Let me know if you have any questions!

**Comment by RamboV** [comment]
> Hi @RamboV!
> 
> Your PR is approved and merged. Please update your submission in ISV Studio by using the latest commit ID.
> 
> Let me know if you have any questions!

Hello @v-EgorKozhadei  @Amjed-Ayoub 
Thanks, In this pull request we missed the properties files update, I have raised one more PR,  https://github.com/microsoft/PowerPlatformConnectors/pull/1779, Please kindly review this and merge. Thanks in Advance. Please let me know if any issues.

**Pull request merged**